datasets/Appenzeller-Herzog_Wilson/process_wilson_dataset.ipynb

```diff
@@ -118,8 +118,9 @@
     "    \n",
     "    cols = list(df)\n",
     "    label_cols = ['label_included', 'label_abstract_screening']\n",
-    "    df = df[list(set(cols) - set(label_cols)) + label_cols]\n",
-    "    return df[list(set(cols) - set(label_cols)) + label_cols]\n",
+    "    cols.sort()\n",
+    "    cols = [elem for elem in cols if elem not in label_cols]\n",
```

Comment:
Good catch!

I would like to skip the `sort()` such that we can preserve the order.